web/static/js/components/alert.jsx

```diff
@@ -10,6 +10,11 @@ Modal.defaultStyles.content.zIndex = 2
 Modal.defaultStyles.overlay.zIndex = 2
 
 export class Alert extends Component {
+
+  componentWillMount() {
```

Comment:
does this need to be set in `componentWillMount`? could we set it after the `zIndex` assignments on line 9 and 10?

Reply:
also, i think the app element might be selected via `.react-root`, rather than body, as that's the div that our react app mounts to

Reply:
Yes, we can set it after the `zIndex` and it works.

So I tried setting it to `.react-root` but the `alert-tests` fail, and I'm not entirely sure why. Example of one failure:

```
Alert component
       "before each" hook for "renders the headerText in a header text component":
     Error: react-modal: No elements were found for selector .remote-root.
```